### DISCUSSION TITLE
Extract images from PDF

### DISCUSSION THREAD

**Discussion by gusye1234** [Unstructured-IO/unstructured#1213] [Q&A]
Hi there!
I'm looking for a solution to extract images from PDF and tried tools like `pymupdf`. I looked through the Document of `unstructured` and it seems didn't support getting images out of PDF. I only saw something like `infer_table_structure `.

Did I miss anything or `unstructured` just currently doesn't support that?

**Comment by ericfeunekes** [comment]
Pretty sure there's an argument to specify where you want to output images and they will be saved to that folder.
